tests/sentry/digests/backends/test_redis.py

```diff
@@ -107,3 +107,14 @@ def test_missing_record_contents(self):
         # contents were merged back into the digest.
         with backend.digest('timeline', 0) as records:
             assert set(records) == set([record_2])
+
+    def test_large_digest(self):
+        backend = RedisBackend()
+
+        n = 16384
```

Comment:
https://github.com/antirez/redis/blob/d680eb6dbdf2d2030cb96edfb089be1e2a775ac1/deps/lua/src/luaconf.h#L439-L446

In the case where the items list was much lower, I wonder if something else was influencing the current size of the stack.